extensions/readwise/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Readwise Changelog
+
+## [v1.0.0] - 2022-03-25
+
+### Features
+
+- Initial release with highlights and library search
```

Comment:
```suggestion
## [v1.0.0] - 2022-03-25

- Initial release with highlights and library search
```

Reply:
Thanks, I will update it. The reason why I added is that the docs mention this:

> You can use [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) to help you format your changelog correctly

And I use this format as well, but with slightly different wording. But I'll change it, works for me.